### ISSUE TITLE
Feature Request: add evolve function to run every 30 minutes or so

### ISSUE THREAD

**Issue by stevecaso** [NecronomiconCoding/NecroBot#677]
Run evolve pokemons then transfer every 30 mins to clean inventory

**Comment by jazzermertize** [comment]
Yea im waiting for that ! Would be great to implement that since it only evolves in the beginning!

**Comment by Rawrlex** [comment]
Yes, evolve check with candies before transfer would be nice. At least for the pokemons in the "PokemonsToEvolve" list.

**Comment by OverCraftX** [comment]
It will be cool to have a file with pokemons to evolve before transfer. To evolve all pidgys, caterpies, weedles and rattata and burn that candies into exp.

I have to manually addthem at ConfigPokemonsToKeep.ini and then run the bot for 1 hour to get some pidgys to evolve. An autosystem will be appreciated.

**Comment by ShawnHardwick** [comment]
Pokemon evolves, transfers, and recycling should occur every 5th pokestop.

See line 99 of FarmPokestopsTask.cs
